src/components/Drawer/Drawer.tsx

```diff
@@ -142,7 +142,7 @@ export const Drawer = ({
   useEffect(() => {
     function handleOnClickOutside(e: MouseEvent) {
       if (
-        isActive &&
```

Comment:
for my understanding: did this not work because when "handleOnClickOutside" is triggered, `isActive` is `false`, so the if is bypassed?

Reply:
When the button to open the drawer is clicked, `isActive` is set to `true` to open the drawer, then the `handleOnClickOutside` event occurs since the button is outside and immediately closes the drawer after... `activeFocus` is toggled to `true` after `handleOnClickOutside` is triggered so won't close the drawer.
previously:
1. `isActive` is set to true
2. since `isActive` is true, `handleOnClickOutside` closes the drawer

now:
1. `isActive` is set to true
2. `activeFocus` is false for now
3. `handleOnClickOutside` doesn't close drawer
4. `activeFocus` is set to true